tests/test_analysis.py

```diff
@@ -0,0 +1,95 @@
+import covasim as cv
+import numpy as np
+
+
+# Suite of tests to test basic functionality of the analysis.py file
+# Runtime: 5.004 seconds
+def test_analysis_snapshot():
+    sim = cv.Sim(analyzers=cv.snapshot('2020-04-04', '2020-04-14'))
+    sim.run()
+    snapshot = sim['analyzers'][0]
+    people1 = snapshot.snapshots[0]            # Option 1
+    people2 = snapshot.snapshots['2020-04-04'] # Option 2
+    people3 = snapshot.get('2020-04-14')       # Option 3
+    people4 = snapshot.get(34)                 # Option 4
+    people5 = snapshot.get()                   # Option 5
+    # people3 = []  # uncomment to verify error

```

Comment:
You can use `with pytest.raises(Exception)` for this, but would suggest just removing

Reply:
This is a different thing- this is just to make sure that the test itself fails when it should (make sure that when the test fails, we see the proper assertion).  Perhaps we can change the comment:
```
uncomment to see assertion raised
```

Reply:
Ah got it -- well, maybe we can remove this anyway if we remove the `assert`